README.md

```diff
@@ -73,6 +73,9 @@ When testing, you can run the app with `python runserver.py`. In production,
 you should use something like [Gunicorn][gunicorn] to serve this. You can run
```

Comment:
Add in the paragraph above (that talks about postgres) that redis is also required if you want to send emails.